src/admin-views/conditional_content/black-friday.php

```diff
@@ -1,36 +1,37 @@
 <?php
 /**
- * Old Template for Black Friday Promo.
+ * Black Friday Conditional template.
  *
- * @since 4.14.7
- * @var string $background_image - the url of the background image to use
- * @var string $branding_logo - the url of the TEC branding logo
- * @var string $button_link - the url the button links to
+ * @since TBD
+ *
+ * @var string $image_src Where the image is located.
+ * @var string $link      Where the image should link to.
+ * @var string $nonce     The nonce for the dismiss button.
+ * @var string $slug      The slug for the dismiss button.
  */
 
+$year = date_i18n( 'Y' );
+/* translators: %1$s: Black Friday year */
```

Comment:
```suggestion
/* translators: %1$s: Black Friday sale year (numeric) */
```
Just clearly indicate it's a number